typhos/tweakable.py

```diff
@@ -0,0 +1,88 @@
+"""
+Tweakable value widget.
+
+Variety support pending:
+- everything
+"""
+import logging
+
+import qtpy
+from qtpy import QtCore
+
+from . import utils, variety
+
+logger = logging.getLogger(__name__)
+
+
+@variety.uses_key_handlers
+@variety.use_for_variety_write('scalar-tweakable')
+class TyphosTweakable(utils.TyphosBase):
+    #  TODO rearrange package: widgets.TyphosDesignerMixin):
```

Comment:
We should transition any TODOs over to GH issues

Reply:
OK, if you agree with them as-is, I'll open up issues with further details.

Reply:
Issues created. TODOs remain in place, but can be removed as the issues are resolved.